### ISSUE TITLE
generating depth map using velodyne lidar when making custom dataset

### ISSUE THREAD

**Issue by Jinho-P** [nianticlabs/monodepth2#193]
Hi,

I'm considering why acuracy is decreasing and evaluate metric (like abs_rel, log_rms) is increasing in training time. Then, I suspects gt_depth using lidar data.  

In kitti_dataset.py, there is a function for making GT depth map as 'generate_depth_map()' and
for resizing depth map using 'self.full_res_shape'.
Also in 'generate_depth_map()' function, it uses "S_rect_02" info in calib_cam_to_cam.txt.
When I makes resized image data and corresponding calibration txt file, I set "S_rect_02" to resized image resolution not original image resolution. 

1. However for making good gt depth map, "S_rect_02" value has a original image resolution...Is it right?   

2. Also, should I change below code in trainer.py file?
```
# garg/eigen crop
crop_mask = torch.zeros_like(mask)
crop_mask[:, :, 153:371, 44:1197] = 1
mask = mask * crop_mask
```

3. what does means "self.full_res_shape" in kitti_dataset.py ?
if I has a three sizes (original image resolution, resized image resolution used training, network input size using options --width & --height), what is self.full_res_shape in these?

**Comment by mrharicot** [comment]
Hi,
1. That's correct.
2. This code is to to match the crop used by Eigen and then Garg for their respective papers. It is here solely to reproduce their evaluation settings.
3. The projection matrix itself doesn't give us the original full resolution image size, this is what this parameter is.
I hope this helps!

**Comment by Jinho-P** [comment]
Thanks for your reply!!!
